### PR TITLE
Feature/35672

### PR DESCRIPTION
**Pull request by szafran89**
https://github.com/SnowdogApps/magento2-alpaca-components/issues/108

### CODE REVIEW

**Comment by szafran89** [comment]
@anqaka I've removed a lot of js code related with popup module. IMO it's useless.

File: components/03-modules/header/_header.scss [hunk 1]
```diff
@@ -13,7 +13,7 @@ $header__icon-color            : $gray-dark !default;
         display: flex;
         flex-flow: row wrap;
         @include mq($screen-xl) {
-            margin: 2px 0;
+            margin: 4px 0;
```
Comment (anqaka) [inline components/03-modules/header/_header.scss]: shouldn't we move it to component variable?

File: components/03-modules/minicart-content/minicart-content.hbs [hunk 2]
```diff
@@ -1,47 +1,28 @@
...
+    <div class="minicart-content__actions">
+        {{ render '@button--secondary' checkout }}
+    </div>
+    <div class="minicart-content__products-list">
+        {{ render '@minicart-product' }}
+        {{ render '@minicart-product' }}
+    </div>
+    <div class="minicart-content__actions">
+        <a href="#" class="minicart-content__edit">
+            View and edit cart
```
Comment (anqaka) [inline components/03-modules/minicart-content/minicart-content.hbs]: shouldn't the text and link be set in config file for component?

File: components/03-modules/minicart-product/_minicart-product.scss [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+            &:focus,
+            &:hover {
+                fill: $minicart-product__action-icon-color--active;
+            }
+        }
+
+        &:focus,
+        &:hover {
+            background-color: $minicart-product__action-button-bg;
+            border: 0;
```
Comment (anqaka) [inline components/03-modules/minicart-product/_minicart-product.scss]: there is no effect on hover fot those buttons?

File: components/03-modules/minicart-product/minicart-product.hbs [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+        </h3>
+        {{#if configurable }}
+            {{ render '@dropdown-list' attributes }}
+        {{/if}}
+        <div class="minicart-product__price">
+            {{ productPrice }}
+        </div>
+        <div class="minicart-product__bottom">
+            <div class="minicart-product__qty">
+                Qty: 32
```
Comment (anqaka) [inline components/03-modules/minicart-product/minicart-product.hbs]: I think this text should be set in config

File: components/03-modules/popup/_popup.scss [hunk 5]
```diff
@@ -16,23 +15,25 @@ $popup__margin                   : 0 !default;
 .popup {
     display: none;
     position: fixed;
-    width: $popup__width;
-    max-width: $popup__max-width;
+    right: 0;
+    left: auto;
+    width: 100%;
```
Comment (anqaka) [inline components/03-modules/popup/_popup.scss]: I think we should put this to variable `$popup__width` and for `screen-s` use other variable: `$popup__width--screen-s`. What do you think?

File: components/03-modules/popup/popup.js [hunk 6]
```diff
@@ -60,44 +60,6 @@
...
-        this.popup.classList.add(this.fullWidth);
-        this.popup.style.setProperty('left', 0);
-      }
-      else if (popupContentCoords.right >= this.body.clientWidth) {
-        this.popup.classList.remove(this.fullWidth);
-        this.popup.style.setProperty('right', 0);
-        this.popup.style.setProperty('left', 'auto');
-      }
-    }
-
```
Comment (anqaka) [inline components/03-modules/popup/popup.js]: so now we will be set position of each popup in styles?

Comment (szafran89) [reply components/03-modules/popup/popup.js]: yes

File: components/03-modules/popup/_popup.scss (hunk 5 above)
Comment (szafran89) [reply components/03-modules/popup/_popup.scss]: Changed. Thank you, I didn't remember what we decided about variables and mq.

File: components/03-modules/minicart-product/minicart-product.hbs (hunk 4 above)
Comment (szafran89) [reply components/03-modules/minicart-product/minicart-product.hbs]: done.

File: components/03-modules/minicart-product/_minicart-product.scss (hunk 3 above)
Comment (szafran89) [reply components/03-modules/minicart-product/_minicart-product.scss]: sure, removed.

File: components/03-modules/minicart-content/minicart-content.hbs (hunk 2 above)
Comment (szafran89) [reply components/03-modules/minicart-content/minicart-content.hbs]: moved.

File: components/03-modules/header/_header.scss (hunk 1 above)
Comment (szafran89) [reply components/03-modules/header/_header.scss]: done.

**Pull request merged**